geoips/compare_outputs.py

```diff
@@ -270,21 +274,24 @@ def images_match(output_product, compare_product, fuzz="5%"):
     #     LOG.info('    *** BAD Images do NOT match exactly ***')
     #     LOG.info('    ***************************************')
     #     return False
-    if fullimg_retval != 0:
-        LOG.info("    ***************************************")
-        LOG.info("    *** BAD Images do NOT match exactly ***")
-        LOG.info("    ***   output_product: %s ***", output_product)
-        LOG.info("    ***   compare_product: %s ***", compare_product)
-        LOG.info("    ***   out_diffimg: %s ***", out_diffimg)
-        LOG.info("    ***   exact_out_diffimg: %s ***", exact_out_diffimg)
-        LOG.info("    ***************************************")
+    if fullimg_ret.returncode != 0:
+        LOG.interactive("    ***************************************")
+        LOG.interactive("    *** BAD Images do NOT match exactly ***")
+        LOG.interactive("    ***   output_product: %s ***", output_product)
+        LOG.interactive("    ***   compare_product: %s ***", compare_product)
+        LOG.interactive("    ***   out_diffimg: %s ***", out_diffimg)
+        LOG.interactive("    ***   exact_out_diffimg: %s ***", exact_out_diffimg)
+        LOG.interactive("    ***************************************")
         return False
 
     LOG.info("**Running exact %s", " ".join(exact_call_list))
-    fullimg_retval = subprocess.call(exact_call_list)
+    fullimg_ret = subprocess.run(
```

Comment:
Updated to use `subprocess.run()` rather than `subprocess.call()`. From the documentation for `subprocess.call()`:

> Do not use stdout=PIPE or stderr=PIPE with this function. The child process will block if it generates enough output to a pipe to fill up the OS pipe buffer as the pipes are not being read from.

I also piped stderr to stdout and added this output to `LOG.debug()`.